package.json

```diff
@@ -47,10 +50,16 @@
     "@types/react-router-dom": "^5.3.2",
     "@typescript-eslint/eslint-plugin": "^5.14.0",
     "@typescript-eslint/parser": "^5.14.0",
+    "babel-loader": "^8.2.1",
+    "babel-plugin-transform-imports": "1.5.1",
     "cache-loader": "^4.1.0",
     "copy-webpack-plugin": "^6.4.1",
     "css-loader": "^6.7.1",
     "cypress": "^11.0.1",
```

Comment:
any reason to use version 11 instead of 13?

Reply:
It was already defined earlier, I had this change in my local now updated